### ISSUE TITLE
Current app.config.sample build fails due to calt setting

### ISSUE THREAD

**Issue by rabidllama** [GIScience/openrouteservice#547]
In the `app.config.sample` file, for car and hgv profiles there is the line 
```json
"lmsets": "highways,tollways;highways;tollways;country_193;allow_all"
```
However, the country borders extended storage is not turned on by default, and so the `country_193` part causes the build process to fail.

**Comment by rabidllama** [comment]
see #531 and #545

**Comment by rabidllama** [comment]
Fixed in Release 5.0.2
